src/renderers/shared/stack/reconciler/ReactComponentEnvironment.js

```diff
@@ -7,12 +7,21 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactComponentEnvironment
+ * @flow
  */
 
 'use strict';
 
 var invariant = require('invariant');
 
+type ReplaceNodeWithMarkup = (node: HTMLElement, markup: string) => void;
+type ProcessChildrenUpdates = (instance: mixed, updates: mixed) => void;
```

Comment:
When ReactInstance and Updates will be implemented, need to go back here and properly type them